src/aics-image-viewer/components/App/types.ts

```diff
@@ -1,8 +1,48 @@
 import { View3d, Volume, ImageInfo } from "@aics/volume-viewer";
-import { ImageType, ViewMode } from "../../shared/enums";
+import { ImageType, RenderMode, ViewMode } from "../../shared/enums";
+import { PerAxis } from "../../shared/types";
 import { ColorArray } from "../../shared/utils/colorRepresentations";
 import { ChannelGrouping, ChannelState, ViewerChannelSettings } from "../../shared/utils/viewerChannelSettings";
 
+type ControlNames =
+  | "alphaMask"
+  | "autoRotateButton"
+  | "axisClipSliders"
+  | "brightnessSlider"
+  | "colorPicker"
+  | "colorPresetsDropdown"
+  | "densitySlider"
+  | "levelsSliders"
+  | "interpolationControl"
+  | "saveSurfaceButtons"
+  | "fovCellSwitchControls"
+  | "viewModeRadioButtons"
+  | "resetCameraButton"
+  | "showAxesButton"
+  | "showBoundingBoxButton";
+/** Show/hide different elements of the UI */
+export type ShowControls = { [K in ControlNames]: boolean };
+
+/** Global (not per-channel) viewer state which may be changed in the UI */
+export interface GlobalViewerSettings {
+  viewMode: ViewMode;
+  renderMode: RenderMode;
+  imageType: ImageType;
+  showAxes: boolean;
+  showBoundingBox: boolean;
+  boundingBoxColor: ColorArray;
+  backgroundColor: ColorArray;
+  autorotate: boolean;
+  maskAlpha: number; // props.viewerConfig.maskAlpha || ALPHA_MASK_SLIDER_3D_DEFAULT,
```

Comment:
can ditch these comments at the end of the lines?

Reply:
A lot of the symbolic constants they reference may be a bit obsolete too, and could instead be refactored a bit into a single default props object in constants.ts